src/mochiweb_http.erl

```diff
@@ -273,6 +271,8 @@ range_skip_length_test() ->
     %% invalid ranges
     ?assertEqual(invalid_range,
                  range_skip_length({-1, 30}, BodySize)),
+    ?assertEqual({0, BodySize},
```

Comment:
There are already cases for `Size < End` around line 258, this case should move to there.